README.md

```diff
@@ -54,3 +54,7 @@ Initial release of ...
 * [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
 
 **Enjoy!**
+
+Don't complete this PR.  I just want to see what it looks like via the email reflector.  
```

Comment:
I wanted to add a single comment without a review but this comment should also have context of the line i added the comment to.  Lets see.

Reply:
That should be better now